### PR TITLE
[DEVX-1067] Add docs for playwright project

### PR DESCRIPTION
**Pull request by tianfeng92**
### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [ ] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/testrunner-toolkit/configuration/playwright.md [hunk 1]
```diff
@@ -429,6 +430,17 @@ The version of Playwright that is compatible with the tests defined in this file
...
 ---
 
+### `configFile`
+<p><small>| OPTIONAL | STRING |</small></p>
+
+The designated `playwright` configuration file. `saucectl` determines related files based on the location of the config file. It can be typescript and TypeScript and JavaScript both.
+If it's not set, `saucectl` will pick default playwright config file(`playwright.config.ts` or `playwright.config.js`).
+
+```yaml
+  configFile: config.ts
```
[the remark this answers is not in the document]
Comment (tianfeng92) [reply docs/testrunner-toolkit/configuration/playwright.md:440]: It should be a relative path and under `rootDir`

File: docs/testrunner-toolkit/configuration/playwright.md [hunk 2]
```diff
@@ -540,13 +552,14 @@ A parent property that details any additional parameters you wish to set for the
...
       slowMo: 1000
+      project: "project name"
 ```
 
 __Description__: This field is for specific test run parameters, for example:
 * `browserName` ( *string* ) : the browser in which to run tests
 * `headful` ( *boolean* ) : whether to run browsers in headless mode
 * `sloMo` ( *int* ) : whether to implement artificially slow load times in milliseconds
-
+* `project` ( *string* ): the playwright project name
```
[the remark this answers is not in the document]
Comment (tianfeng92) [reply docs/testrunner-toolkit/configuration/playwright.md:562]: sure.

File: docs/testrunner-toolkit/configuration/playwright.md [hunk 3]
```diff
@@ -429,6 +430,17 @@ The version of Playwright that is compatible with the tests defined in this file
 ```
 ---
 
+### `configFile`
+<p><small>| OPTIONAL | STRING |</small></p>
+
+The designated `playwright` configuration file. `saucectl` determines related files based on the location of the config file. Supports both TypeScript and JavaScript files.
+If it's not set, `saucectl` will pick default playwright config file(`playwright.config.ts` or `playwright.config.js`).
```
Comment (sweeneyskirt-sl) [inline docs/testrunner-toolkit/configuration/playwright.md:437]:
```suggestion
The path (relative to `rootDir`) to your Playwright configuration file. `saucectl` determines related files based on the location of this config file. Supports both TypeScript and JavaScript files.
If it's not set, `saucectl` defaults to `playwright.config.ts` or `playwright.config.js`.
```

**Review by sweeneyskirt-sl — CHANGES_REQUESTED** [review]
Just one more minor edit. Thanks, @tianfeng92 !

**Review by sweeneyskirt-sl — APPROVED** [review]

**Pull request merged**